lib/insights_cloud/async/vmaas_reposcan_sync_task.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'vmaas_reposcan_sync_task_base'
+
+module InsightsCloud
+  module Async
+    # Triggers VMaaS reposcan sync when repositories are synced
+    class VmaasReposcanSyncTask < VmaasReposcanSyncTaskBase
+      def plan(repo, *_args)
+        return unless repo.is_a?(::Katello::Repository) && ForemanRhCloud.with_iop_smart_proxy?
+
+        plan_self
+      end
+
+      def self.subscribe
+        return unless defined?(::Katello)
```

Comment:
Does something break without this? The other two `def self.subscribe`s in rh_cloud don't have it.